Package.swift

```diff
@@ -6,28 +6,27 @@ import class Foundation.ProcessInfo
 
 let env = ProcessInfo.processInfo.environment
 
-guard let llvmHeaderPath = env["SWIFT_LLVM_BINDINGS_PATH_TO_LLVM_HEADERS"] else {
-  fatalError("please pass an environment variable to swift-package: " +
-               "SWIFT_LLVM_BINDINGS_PATH_TO_LLVM_HEADERS " +
-               "(e.g. swift/llvm-project/llvm/include)")
-}
-let llvmModuleMapPath = "\(llvmHeaderPath)/llvm/module.modulemap"
-guard let llvmGeneratedHeaderPath = env["SWIFT_LLVM_BINDINGS_PATH_TO_LLVM_GENERATED_HEADERS"] else {
-  fatalError("please pass an environment variable to swift-package: " +
-               "SWIFT_LLVM_BINDINGS_PATH_TO_LLVM_GENERATED_HEADERS " +
-               "(e.g. swift/build/Ninja-DebugAssert/llvm-macosx-arm64/include)")
-}
+func getLLVMSwiftSettings() -> [SwiftSetting]? {
+    let env = ProcessInfo.processInfo.environment
+    guard let llvmHeaderPath = env["SWIFT_LLVM_BINDINGS_PATH_TO_LLVM_HEADERS"] else {
+        return nil
```

Comment:
Can we print some kind of a warning if the env variable is not defined? That would make it easier to diagnose build errors for packages that have a (possibly transitive) dependency on the bindings.

Reply:
Sure, that's addressed now

Reply:
Awesome, thanks!